platform/viewer/src/routes/StandaloneRouting.js

```diff
@@ -96,53 +83,34 @@ class StandaloneRouting extends Component {
 
       // Remove ? prefix which is included for some reason
       search = search.slice(1, search.length);
-
       const query = qs.parse(search);
-      const studies = await StandaloneRouting.parseQueryAndFetchStudies(query);
-
-      studyMetadataManager.purge();
-
-      // Map studies to new format, update metadata manager?
-      const uniqueStudyUids = new Set();
-      const updatedStudies = studies.map(study => {
-        const studyMetadata = new OHIFStudyMetadata(
-          study,
-          study.studyInstanceUid
-        );
-        const sopClassHandlerModules =
-          extensionManager.modules['sopClassHandlerModule'];
 
-        study.displaySets =
-          study.displaySets ||
-          studyMetadata.createDisplaySets(sopClassHandlerModules);
-        studyMetadata.setDisplaySets(study.displaySets);
+      const {
+        server,
+        studyInstanceUids,
+        seriesInstanceUids,
+      } = await StandaloneRouting.parseQueryAndRetrieveDICOMWebData(query);
 
-        // Updates WADO-RS metaDataManager
-        updateMetaDataManager(study);
-
-        studyMetadataManager.add(studyMetadata);
-        uniqueStudyUids.add(study.studyInstanceUid);
-
-        return study;
-      });
-
-      this.studyInstanceUids = Array.from(uniqueStudyUids);
-      this.setState({ studies: updatedStudies });
+      this.setState({ server, studyInstanceUids, seriesInstanceUids });
     } catch (error) {
-      this.setState({ error });
+      this.setState({ error: error.message });
     }
   }
 
   render() {
-    if (this.state.error) {
-      return <div>Error: {JSON.stringify(this.state.error)}</div>;
-    } else if (!this.state.studies) {
-      return <div>Loading...</div>;
+    const message = this.state.error ? `Error: ${JSON.stringify(this.state.error)}` : 'Loading...';
+    if (this.state.error || !this.state.server) {
```

Comment:
This `!this.state.server` part of the statement is not correct and breaks the `checks PT/CT json url study route` test. You also need to be able to load it with: https://viewer.ohif.org/viewer?url=https://s3.eu-central-1.amazonaws.com/ohif-viewer/JSON/PTCTStudy.json (replace viewer.ohif.org with localhost:3000).

Otherwise the PR looks good!